hedgehog.cabal

```diff
@@ -34,15 +34,17 @@ source-repository head
 library
   build-depends:
       base                            >= 3          && < 5
-    , base64-bytestring               >= 1.0        && < 1.1
+    , ansi-terminal                   >= 0.6        && < 0.7
```

Comment:
Yeah I think it's probably worth waiting until there is a concrete burden before doing this